index.html

```diff
@@ -270,14 +270,14 @@ <h3>SUBSCRIBE</h3>
 
         <div class="vid-list">
           <a href="player.html"
-            ><img class="thumbnail" src="images/thumbnail1.jpg" alt=""
+            ><img class="thumbnail" src="images/thumbnaill.jpg" alt=""
           /></a>
```

Comment:
Kindly try to keep the name of the picture here in thumbnail1.jpg or just use the one inside the project.
Thankyou Kely